testdata/baselines/reference/submodule/compiler/assertionFunctionsCanNarrowByDiscriminant.js.diff

```diff
@@ -7,8 +7,4 @@
 -"use strict";
```

Comment:
I wonder if this very common diff is an easy fix somewhere...

Reply:
This is a script file by strada logic. (no exports/imports, nothing forcing module mode parsing.) In corsa, the module transforms add `use strict` if needed, but in strada, the `ts` transform does (via `visitLexicalEnvironment` in `visitSourceFile`) so it's done for script files. Obviously the module transforms don't touch script files, so they don't add the prologue.